### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
I added the Home page screen.

-This page contains a Navigation bar, header section Feature section, footer section.
-CSS related to the home page screen you can find in style.css file and HTML you find in Index.html file.
-For the Navigation bar I used my component library. the modification for navbar I added in modification_nav_CL.css file
-for the feature section, I used a feature card from my component library
-for log-in BOX you can click on the profile icon so the login box opens.(maybe I will modify it later).

Please review my code and Approve it.
Thanks.

**screenshots**
img1
img2
img3
img4

### CODE REVIEW

**Review by psychoactiv — APPROVED** [review]

**Comment by psychoactiv** [comment]
Code looks good to merge

File: style.css [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+.features .box-container{
+  margin: 2rem;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(30rem, 1fr));
+  gap:3rem;
+}
+
+.groce-footer{
+  background: #fff;
```
Comment (shaksham08) [inline style.css]: Use color variable

**Review by shaksham08 — APPROVED** [review]
The code looks good. Just added one comment

File: style.css (hunk 1 above)
Comment (kotalwar-krishna) [reply style.css]: ok

**Pull request merged**